packages/app/src/cli/utilities/developer-platform-client/app-management-client.ts

```diff
@@ -575,11 +576,14 @@ export class AppManagementClient implements DeveloperPlatformClient {
         configurationRegistrations.push(registration)
       } else {
         extensionRegistrations.push(registration)
+        if (mod.specification?.options?.managementExperience === 'dashboard') {
+          dashboardManagedExtensionRegistrations.push(registration)
+        }
```

Comment:
Dashboard extensions are added to both the normal array and the dashboard array?

Reply:
I thought Partners was doing that, but just checked that extensions only belong to one group, so I've updated this. Thanks!

It was not affecting the deploy, because it only uses dashboard managed extensions, but `import-extensions` was showing duplicated entries with App Management because of that 😅